variants/frontend-stimulus/hello_controller.js

```diff
@@ -0,0 +1,12 @@
+import { Controller } from '@hotwired/stimulus';
+
+// To see this controller in action, add the following to the DOM:
+//
+//   <div data-controller="hello"></div>
+
+// eslint-disable-next-line import/no-anonymous-default-export
+export default class extends Controller {
```

Comment:
You might already be planning to change this, but fwiw there is no good reason to not explicitly name this class given its a default export, and omitting it only makes it harder to find and debug (because JS will pick the name based on the variable its assigned to which is probably going to be something generic)

Reply:
All the stimulus examples from the docs leave the name off e.g. [here](https://stimulus.hotwired.dev/handbook/hello-stimulus#controllers-bring-html-to-life). I think this is to give you one less step when you are renaming a controller. I'm open to including the name in our example, especially if it helps IDEs do more for us but we should be aware that controllers probably won't follow it unless we enforce it somehow

Reply:
> unless we enforce it somehow

That is what `import/no-anonymous-default-export` is trying to do

Reply:
Fair enough. I'll name it.